dbinteract.py

```diff
@@ -101,6 +102,24 @@ def fetch_timezone(guild_id):
     return timezone_minutes
     
 
+def sort_deadlines(ans):
+    #sort deadlines using selection sort
+    for i in range(len(ans)):
+        for j in range(i+1, len(ans)):
+            if ans[j][1] < ans[i][1]:
+                ans[i] = list(ans[i])
+                ans[j] = list(ans[j])
+                
+                temp = ans[j]
+                ans[j] = ans[i]
+                ans[i] = temp
+                
+                ans[i] = tuple(ans[i]) 
+                ans[j] = tuple(ans[j]) 
```

Comment:
this works fine but converting to list just to convert back to tuple is redundant. please remove that

Reply:
we have to convert to list to change the elements. Can't assign the elements in a tuple.
The workaround is changing the function to return list everywhere